src/config.ts

```diff
@@ -0,0 +1,83 @@
+import { Environment, ENVIRONMENTS } from './util';
+import { MessengerType } from './messenger';
+
+/**
+ * Muxy production API URL.
+ * @ignore
+ */
+const API_URL = 'https://api.muxy.io';
+
+/**
+ * Muxy sandbox API URL.
+ * @ignore
+ */
+const SANDBOX_URL = 'https://sandbox.api.muxy.io';
+
+/**
+ * Localhost for testing purposes.
+ * @ignore
+ */
+const LOCALHOST_URL = 'http://localhost:5000';
+
+/**
+ * Muxy staging API URL.
+ * @ignore
+ */
+const STAGING_URL = 'https://staging.api.muxy.io';
```

Comment:
Should we go ahead and remove the hardcoded references to staging in this PR, since I assume we'll be adding them back in with the programmatic environment checks?